service/src/main/java/io/camunda/service/GroupServices.java

```diff
@@ -123,18 +123,18 @@ public CompletableFuture<GroupRecord> deleteGroup(final long groupKey) {
   }
 
   public CompletableFuture<GroupRecord> assignMember(
-      final long groupKey, final long memberKey, final EntityType memberType) {
+      final String groupId, final String username, final EntityType memberType) {
```

Comment:
🔧 Same comment about using a DTO as in the other PRs

Reply:
There is not DTO, groupId and username are coming from the path

Reply:
That's interesting. We also don't validate the user input it seems. Personally I'd say we need to adhere to the architecture we have:

1. Request comes in through the controller
2. Controller goes to the request mapper which will validate the request and map it to an object (group id and username valid and in correct format)
3. Object gets passed along to the service who will turn it into a broker request.

Anyway I'll approve the PR because it's unrelated here.

Reply:
I'll create a separate issue for it, but this will affect multiple endpoints (tenant as well)